{{cookiecutter.project_folder}}/.gitignore

```diff
@@ -94,27 +95,52 @@ ENV/
 # Rope project settings
 .ropeproject
 
+# PyCharm project settings
+.idea/
+
 # mkdocs documentation
 /site
 
 # mypy
 .mypy_cache/
 
-# ==============================================================================
-# {{ cookiecutter.project_name }}
-# ==============================================================================
+# ------------------------------------------------------------------------------
+# compas
+# ------------------------------------------------------------------------------
+
+dist/docs
+
+generated/
 
 *.3dmbak
 *.rhl
-*.rui_bak
+
+libs/**
+!libs/PLACEHOLDER
 
 temp/**
 !temp/PLACEHOLDER
 
-.DS_Store
+src/.build/**
+
+data/bunny
+data/models
+data/ctralie
+*.tar.gz
+
+data/models
+data/ctralie
```

Comment:
maybe just `data`? sample data that is part of the distribution should be included inside the package source in my opinion

Reply:
sure done!